### PR TITLE
Separate the pulp_installer CI tests: release vs source

### PR DESCRIPTION
**Pull request by mikedep333**
Required PR: https://github.com/pulp/pulp_installer/pull/269
    
re: #6535
    
[noissue]

### CODE REVIEW

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -38,18 +57,47 @@ jobs:
...
+          find ./ -type f -exec sed -i -e 's/travis_fold:end:$containerlog/::endgroup::/g' {} \;
+          find ./ -type f -exec sed -i -e 's/.\/.travis\/check_commit.sh/#.\/.travis\/check_commit.sh/g' {} \;
+      - name: Before Install
+        run: |
+          .travis/before_install.sh
+          echo "::set-env name=TEST::${{ matrix.env.TEST }}"
+          cd ..
+          find ./ -type f -exec sed -i -e 's/timeout 90 bash/timeout 120 bash/g' {} \;
+          find ./ -type f -exec sed -i -e 's/\/usr\/local\/bin\/kubectl/$(which kubectl)/g' {} \;
+          find ./ -type f -exec sed -i -e 's/# containers are not running yet./sudo cp -i \/etc\/rancher\/k3s\/k3s.yaml $HOME\/.kube\/config/g' {} \;
```
Comment (fao89) [inline .github/workflows/pull_request.yaml]: these changes were for `k3s`

File: .github/workflows/pulp_ci.yaml [hunk 2]
```diff
@@ -35,6 +41,29 @@ jobs:
...
+          find ./ -type f -exec sed -i -e 's/.\/.travis\/check_commit.sh/#.\/.travis\/check_commit.sh/g' {} \;
+      - name: Before Install
+        run: |
+          .travis/before_install.sh
+          echo "::set-env name=TEST::${{ matrix.env.TEST }}"
+          cd ..
+          find ./ -type f -exec sed -i -e 's/timeout 90 bash/timeout 120 bash/g' {} \;
+          find ./ -type f -exec sed -i -e 's/\/usr\/local\/bin\/kubectl/$(which kubectl)/g' {} \;
+          find ./ -type f -exec sed -i -e 's/# containers are not running yet./sudo cp -i \/etc\/rancher\/k3s\/k3s.yaml $HOME\/.kube\/config/g' {} \;
+          cd pulp_rpm_prerequisites
```
Comment (fao89) [inline .github/workflows/pulp_ci.yaml]: I believe we don't need it here, as the `Required PR` thing is just for PRs

**Review by fao89 — COMMENTED** [review]
I prefer the way you handled Required PR on galaxy installer than bringing plugin_template here:
https://github.com/fabricio-aguiar/galaxy_ng_prerequisites/pull/3/files

File: .github/workflows/pull_request.yaml (hunk 1 above)
Comment (mikedep333) [reply .github/workflows/pull_request.yaml]: Yeah, originally I figured I'd purposely leave them there to make the delta against plugins smaller, but code clarity is important to, so I just removed them.

File: .github/workflows/pulp_ci.yaml (hunk 2 above)
Comment (mikedep333) [reply .github/workflows/pulp_ci.yaml]: I modified the script (and moved it from .travis/ to .github/)

**Review by fao89 — APPROVED** [review]
Thank you @mikedep333

**Comment by mikedep333** [comment]
> I prefer the way you handled Required PR on galaxy installer than bringing plugin_template here:

I think for one only 1 referenced repo, that make sense. I was mostly focused on avoiding writing duplicate code.

At this point, I just want to merge. I am likely to rework this section of code later when I implement another task in this story for checking out pulp_installer branches. (e.g., pulp_rpm_prerequisites 3.3 branch would use pulp_installer 3.3 branch, rather than setting the pulp_version variable in pulp_installer master.)

**Pull request merged**